WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/Addresses/EditOrderAddressFormViewModelTests.swift

```diff
@@ -701,6 +701,25 @@ final class EditOrderAddressFormViewModelTests: XCTestCase {
         assertEqual(analyticsProvider.receivedProperties.first?["subject"] as? String, "billing_address")
     }
 
+    func test_view_model_when_customerSelectedFromSearch_then_tracks_orderCreationCustomerAdded() {
+        // Given
+        let analyticsProvider = MockAnalyticsProvider()
+        let viewModel = EditOrderAddressFormViewModel(order: Order.fake(), type: .billing, analytics: WooAnalytics(analyticsProvider: analyticsProvider))
+        let customer = Customer.fake().copy(
+            email: "scrambled@scrambled.com",
+            firstName: "Johnny",
+            lastName: "Appleseed",
+            billing: sampleAddressWithEmptyNullableFields(),
+            shipping: sampleAddressWithEmptyNullableFields()
+        )
+
+        // When
+        viewModel.customerSelectedFromSearch(customer: customer)
+
+        // Then
+        assertEqual(analyticsProvider.receivedEvents, [WooAnalyticsStat.orderCreationCustomerAdded.rawValue])
```

Comment:
I didn't thought about it, if some other event sneaks in while we're waiting from our `customerSelectedFromSearch()` to come back, the test would fail indeed. I changed the test to confirm if `receivedEvents` contains it instead.

```
// Then
- assertEqual(analyticsProvider.receivedEvents, "order_creation_customer_added")
+ XCTAssert(analyticsProvider.receivedEvents.contains("order_creation_customer_added"))
```

Fixed here: 169094db25cdda1074fbfa2ea9804851f6cb38d0